packages/strapi-plugin-email/services/Email.js

```diff
@@ -1,12 +1,42 @@
 'use strict';
 
+const _ = require('lodash');
+
 const getProviderConfig = () => {
   return strapi.plugins.email.config;
 };
 
+const send = async options => {
+  return strapi.plugins.email.provider.send(options);
+};
+
+/**
+ * fill subject, text and html using lodash template
+ * @param {object} emailOptions - to, from and replyto...
+ * @param {object} emailTemplate - object containing attributes to fill
+ * @param {object} data - data used to fill the template
+ * @returns {{ subject, text, subject }}
+ */
+const fillEmailOptions = (emailOptions = {}, emailTemplate = {}, data = {}) => {
```

Comment:
I think it would be simpler for the users to call a `sendTemplatedEmail` rather than preparing it then sending it.